go/parquet/pqarrow/schema_test.go

```diff
@@ -36,9 +36,13 @@ func TestGetOriginSchemaBase64(t *testing.T) {
 	origArrSc := arrow.NewSchema([]arrow.Field{
 		{Name: "f1", Type: arrow.BinaryTypes.String, Metadata: md},
 		{Name: "f2", Type: arrow.PrimitiveTypes.Int64, Metadata: md},
+		{Name: "uuid", Type: pqarrow.NewUUIDType(), Metadata: md},
```

Comment:
in addition to the schema `ToParquet`/`FromParquet` functions. Please add some tests for reading and writing a table / record batches which contain extension types. Such as a simple round trip:

```go
expectedTable := // ... table with extension type column
defer expectedTable.Release()

var buf bytes.Buffer
require.NoError(t, pqarrow.WriteTable(expectedTable, &buf, ....))

actualTable, err := pqarrow.ReadTable(context.Background(), bytes.NewReader(buf.Bytes()), .....)
require.NoError(t, err)
defer actualTable.Release()

assert.True(t, array.TableEqual(expectedTable, actualTable), ...)
```